lib/foreman_rh_cloud/engine.rb

```diff
@@ -52,7 +52,7 @@ def self.register_scheduled_task(task_class, cronline)
             :generate_foreman_rh_cloud,
             'foreman_inventory_upload/reports': [:generate],
             'foreman_inventory_upload/tasks': [:create],
-            'api/v2/rh_cloud/inventory': [:sync_inventory_status, :download_file, :generate_report, :enable_cloud_connector],
+            'api/v2/rh_cloud/inventory': [:get_hosts, :remove_hosts, :sync_inventory_status, :download_file, :generate_report, :enable_cloud_connector],
```

Comment:
I suppose we also need to add 
``` ruby
            'foreman_inventory_upload/missing_hosts': [:remove_hosts],
```
to the "generate" role. (It should be called the admin role, as opposed to the viewer role, but I don't want to change it for backwards compatibility)

Reply:
+1